lib/metrics/prometheus.js

```diff
@@ -34,23 +37,33 @@ class PrometheusMetric {
         }
     }
 
+    _normalize(str) {
+        return String(str).replace( /[^a-z0-9]/g, '_' ) // replace non-alphanumerics
```

Comment:
No need to new String. The string is a String.

Reply:
alphanumeric regex in JS is `\w`

Reply:
From tests, it appears str is not String() (nodejs 13):
```
Uncaught TypeError: str.replace is not a function
      at _normalize (lib/metrics/prometheus.js:41:20)
```